.gitignore

```diff
@@ -391,6 +393,11 @@
 /utils/**/vc_mswudll_x64
 /utils/**/vc_mswuddll
 /utils/**/vc_mswuddll_x64
+/utils/**/*.vcproj*
```

Comment:
I think it would be better to make this one and the following more specific, e.g. `*.vcproj.*.user` and `*.vcxproj.user`, we don't want to accidentally ignore some other files which might be used by MSVS in the future.

I'm also not sure if they should be here at all as they're not wx-specific and I've always recommended adding them (as well as `*.suo` etc) to `$HOME/.gitignore` so that they can be used by all projects. But if people really want to have them here, I guess it's not really a problem...